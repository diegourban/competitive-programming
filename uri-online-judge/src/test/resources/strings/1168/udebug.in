13
0
1
2
3
4
5
6
7
8
9
10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
1
5000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
