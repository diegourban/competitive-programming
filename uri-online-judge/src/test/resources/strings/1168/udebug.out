6 leds
2 leds
5 leds
5 leds
4 leds
5 leds
6 leds
3 leds
7 leds
6 leds
602 leds
2 leds
599 leds
